supplementary/alica_ros_turtlesim/ros1/src/base.cpp

```diff
@@ -23,11 +24,15 @@ Base::Base(ros::NodeHandle& nh, ros::NodeHandle& privNh, const std::string& name
         : spinner(0)
 {
     // Initialize Alica
-    ac = new alica::AlicaContext(AlicaContextParams(name, {path + "/etc/"}, roleset, master_plan, false, agent_id));
+    ac = new alica::AlicaContext(AlicaContextParams(name, {path + "/../alica_turtlesim/etc/", path + "/etc/"}, roleset, master_plan, false, agent_id));
```

Comment:
Do we have two folders?  Or are all plans just from generic alica_turtlesim/etc?

Reply:
we have the shared etc folder in alica_turtlesim, which contains plans like SurroundLeader, and the ros specific etc folder which contains the behaviours WaitForMsg and WairForTrigger

Reply:
I think ros specific behaviors should ideally use the turtle interface as well